vignettes/continuous-integration.Rmd

```diff
@@ -37,6 +37,7 @@ This file configures all the steps required to run `lintr::lint_package()` on yo
 [annotations](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/about-status-checks#types-of-status-checks-on-github)
 along side the status check on GitHub.
 If you want to disable the commenting you can set the environment variable `LINTR_COMMENT_BOT=false`.
+Additionally, if you want the builds to produce an error instead of just a warning, you can set the environment variable `LINTR_ERROR_ON_LINT=true`.
```

Comment:
This should probably explicitly mention that this will kill the R process in case of a lint (produce a non-zero exit code, maybe?)